app/css/global.css

```diff
@@ -26,3 +26,61 @@ body h1 {
 body a {
   color: var(--yellow);
 }
+
+.contenedor{
+  display: grid;
+  grid-template-columns: 3fr 1fr;
+  grid-template-rows: auto;
+}
+
+.item-1 {
+  grid-row: 1; 
+  grid-column: 1;
+}
+
+.item-2 {
+  grid-row: 1; 
+  grid-column: 2;
+}
+
+.item-3 {
+  grid-row: 2; 
+  grid-column: 1;
+}
+
+.item-4 {
+  grid-row: 2; 
+  grid-column: 2;
+}
+
+.contenedor1{
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  grid-template-rows: auto;
+}
+
+.item-2-1 {
+  grid-row: 1; 
+  grid-column: 1;
+}
+
+.item-2-2 {
+  grid-row: 1; 
+  grid-column: 2/4;
+}
+
+.register{
+  font-family: "Soleil";
+  background-color: white;
+}
+
+.img{
+  margin: 15px;
+  align-items: flex-end;
+}
```

Comment:
Split this into another File, register.css or some similar & imported into the styles.css, if you have complications to name the clases use the [BEM Naming convention](http://getbem.com/naming/)